tide/master/rest/RestCommand.cpp

```diff
@@ -62,6 +88,13 @@ bool RestCommand::_fromJSON( const std::string& string )
         put_flog( LOG_INFO, "Error parsing JSON string: '%s'", string.c_str( ));
         return false;
     }
+
+    if( !_takesValue )
```

Comment:
not from the received JSON object, because if the command does not take a parameter then it should not emit any value. Conversly, if it takes a parameters, we can only accept the command if the value is present.